packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Changelog
 
 ## Unreleased
-
-* Added
+* Fixed
+  * Removed roving tabIndex and resolved focus management issues within cells.
+  * Reduced number of unnecessary rerenders in the `terra-worklist-data-grid` component.
```

Comment:
This seems to be a merge conflict

Reply:
I believe that I have resolved the merge conflict issue now.